Kitodo-DataFormat/src/main/java/org/kitodo/dataformat/access/MediaUnit.java

```diff
@@ -0,0 +1,207 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+
+package org.kitodo.dataformat.access;
+
+import java.math.BigInteger;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import org.kitodo.api.dataformat.mets.FLocatXmlElementAccessInterface;
+import org.kitodo.api.dataformat.mets.FileXmlElementAccessInterface;
+import org.kitodo.api.dataformat.mets.UseXmlAttributeAccessInterface;
+import org.kitodo.dataformat.metskitodo.DivType;
+import org.kitodo.dataformat.metskitodo.DivType.Fptr;
+import org.kitodo.dataformat.metskitodo.FileType;
+import org.kitodo.dataformat.metskitodo.Mets;
+
+public class MediaUnit implements FileXmlElementAccessInterface {
+    /**
+     * Each media unit can be available in different variants, for each of which
+     * a media file is available. This is in this map.
+     */
+    private Map<MediaVariant, MediaFile> mediaFiles = new HashMap<>();
+
+    /**
+     * Sequence number of the media unit. The playback order of the media units
+     * when referenced from a structure is determined by this attribute (not by
+     * the order of the references).
+     */
+    private int order;
+
+    /**
+     * A human readable label for the order of this media unit. This need not be
+     * directly related to the order number. Examples of order labels could be
+     * “I, II, III, IV, V,  - , 1, 2, 3”, meanwhile the order would be “1, 2, 3,
+     * 4, 5, 6, 7, 8, 9”.
+     */
+    private String orderlabel;
+
+    /**
+     * Public constructor for a new media unit. This constructor can be used
```

Comment:
I don't get this javadoc. "It can be used with the service loader..."
Is this the only usage, or why are you pointing this out? A constructor can be used in every place where I want a new instance, right?

Reply:
It can only be used via the service loader because the content of the module is not visible from the outside. You can not see that in the code, so if you're looking for references to the constructor, you will not find one. Nevertheless one should not delete it therefore as unused code. That's why.